saleor/discount/migrations/0047_move_codes_to_new_model.py

```diff
@@ -0,0 +1,56 @@
+# Generated by Django 3.2.20 on 2023-08-21 07:23
+
+from django.db import migrations
+
+
+BATCH_SIZE = 5000
```

Comment:
What is the time of single batch update for this batch size? We should add comment here